### ISSUE TITLE
IOS execution, no module named _event

### ISSUE THREAD

**Issue by gunerguney** [kivy/kivy#3966]
Actually the project which was created in pycharm is running successfully, however I could not achieve to run my app in IOS.

https://kivy.org/docs/guide/packaging-ios.html

According to instruction in the link above, I applied all the stepts.

I compiled my app in Xcode successfuly. It starts to run the app in simulator but it gives an error **"no module named _event".**

Could you help me please ?

Regards.
Guney GUNER
guney_guner@hotmail.com

**Comment by cbenhagen** [comment]
This sounds like the cython parts of kivy did not get compiled for some reason. The packaging guide still mentions installing an older cython version. So you could give this a try:

Please `sudo pip install -I Cython==0.23`
and then
````
$ ./toolchain.py clean kivy
$ ./toolchain.py build kivy
$ ./toolchain.py update yourappname-ios
````

And post the logs in a [gist link](https://gist.github.com/)

**Comment by gunerguney** [comment]
No it did not worked

**Comment by tito** [comment]
How did you compile?
What's the *full compilation log*?

**Comment by gunerguney** [comment]
Actually I encountered with this error for solving no module named garden, I could not achieve and removed kivy-ios folder.

Then I appllied all instructions given in  your document, then the same problem occured. It was name of the mapview folder. The standard name of this folder is "garden.mapview"  , I changed the folder name as "garden" (also contains a mapview folder)  and moved it to "kivy/ios/dist/root/python2.7/kivy/" then problem was solved.

I downloaded the futures and requests from

https://pypi.python.org/pypi/futures
https://pypi.python.org/pypi/requests

unzipped both folder and moved to under the kivy-ios/dist/root/python2.7

The problem was solved now my application is running on ios.

Thank you so much.
Regards.
